### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Manage Access Policies_Add_ScreenReader: Voiceover does not read  any status as row added when add and row removed when remove.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#1980]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11YTCS; #DesktopApp; #A11yWCAG2.1;#MAS4.1.3 #VoiceOver; #Mac; 
**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
MAC Version 10.14.6 Beta(18G29g)
**Repro Steps:**
Launch Storage Explorer
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and Select "Create Blob Container" and hit enter.
5. Right Click on newly created blob container and select "Manage Access Policies" and hit enter.
6. Navigate to "Add" control and hit enter.
7. Start VoiceOver and using Tab key and check whether the Voiceover reads any status as row added when add and row removed when remove.
**​Actual:​**
Voiceover does not read  any status as row added when add and row removed when remove.
**Expected:**
VoiceOver should read the status as row added when add and row removed when remove.
**UserImpact:**
Screen reader users unable to know the which dialog opened if it will not read the title.
Recommendation:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
MAS4.1.3:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=13bc100c-7d1a-4d0c-baa4-2e137da441aa
**Attachment for Reference:**
[1980_A11y_AzureToolsStorageExplorer_Manage Access Policies_Add_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3669907/1980_A11y_AzureToolsStorageExplorer_Manage.Access.Policies_Add_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT  Verified the issue on version 1.12.0 and MAC 10.15.3.The issue is no longer reproduced.Hence close this bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
